test/text_overlay/text_overlay_test.exs

```diff
@@ -98,11 +101,49 @@ defmodule TextOverlay.TextOverlayTest do
     Pipeline.stop_and_terminate(pid, blocking?: true)
 
     Helpers.create_ffmpeg_reference(
-      "640x360.h264",
+      "2s_8fps.h264",
       ref_path,
       "drawtext=text='My text':fontcolor=white:box=1:boxcolor=orange:borderw=1:bordercolor=DarkGray:fontsize=35:x=(w-text_w)/2:y=w/100"
     )
 
     Helpers.compare_contents(out_path, ref_path)
   end
+
+  @tag :tmp_dir
+  test "apply text stream on part of the video", %{tmp_dir: tmp_dir} do
+    {in_path, out_path, ref_path} =
+      Helpers.prepare_paths("4s_2fps.h264", "ref-stream.yuv", tmp_dir)
```

Comment:
All the videos have nice,round timestamps hiding potential problems with Ratio. Please add at least one test with 9fps (or something similar that won't divide second without error)